packages/atomic/src/locales.json

```diff
@@ -1113,5 +1113,17 @@
   "query-suggestions-unavailable": {
     "en": "There are no search suggestions.",
     "fr": "Il n'y a pas de suggestions de recherche."
+  },
+  "stars": {
+    "en": "one out of {{max}} stars",
+    "fr": "une étoile sur {{max}}"
+  },
+  "stars_plural": {
```

Comment:
This works well for English and French, but some languages have multiple plural forms (a separate word for 1 vs. 2 vs. many).

Could using [i18next's plural](https://www.i18next.com/translation-function/plurals) features be helpful in this case?


Alternatively, could placing `étoile` at the end of the phrase like in the English example remove the need for a plural key?

Reply:
In this case, having star at the beginning sounds best. It should be easy in the future when we want to add a language with multiple plural forms.